test/exe/Main.hs

```diff
@@ -929,6 +929,48 @@ removeImportTests = testGroup "remove import actions"
             , "module ModuleB where"
             ]
       liftIO $ expectedContentAfterAction @=? contentAfterAction
+  , testSession "remove all" $ do
+      let contentA = T.unlines
+            [ "module ModuleA where"
+            , "staffA :: Integer"
+            , "staffA = 123"
+            , "staffB = True"
+            , "data D = A | B"
+            ]
+      _docA <- createDoc "ModuleA.hs" "haskell" contentA     
+      let contentB = T.unlines
+            [ "module ModuleB where"
+            , "staffC = ()"
+            , "staffD = False"
+            , "data E = F"
+            ]
+      _docB <- createDoc "ModuleB.hs" "haskell" contentB
+      let contentC = T.unlines
+            [ "{-# OPTIONS_GHC -Wunused-imports #-}"
+            , "module ModuleC where"
+            , "import ModuleA (staffA, staffB, D(A, B))"
+            , "import ModuleB (staffC, staffD, E(F))"
```

Comment:
You might be able to simplify this test by using imports from `base` instead of defining auxiliary modules